TransformationSystem/Client/TaskManager.py

```diff
@@ -820,13 +820,25 @@ def _handleRest(self, oJob, paramsDict):
                          transID=transID, method='_handleRest')
           oJob._addJDLParameter(paramName, paramValue)
 
-  def _handleHospital(self, oJob):
+  def _handleHospital(self, oJob, transID):
     """ Optional handle of hospital jobs
     """
     oJob.setInputDataPolicy('download', dataScheduling=False)
-    hospitalSite = self.opsH.getValue("Hospital/HospitalSite", 'DIRAC.JobDebugger.ch')
+
+    hospitalSite = None
+    hospitalCEs = []
+    if "Clinics" in self.opsH.getSections("Hospital")['Value']:
+      clinics = self.opsH.getSections("Hospital/Clinics")['Value']
+      for clinic in clinics:
+        if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/%s/Transformations"%clinic, [])]:
```

Comment:
This line and the next(s), just fix the spacing:
```suggestion
        if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/%s/Transformations" % clinic, [])]:
```